### ISSUE TITLE
Integrity issue with ag_login table

### ISSUE THREAD

**Issue by adamrp** [biocore/american-gut-web#472]
The `email` field (unless I'm mistaken?) should have a unique constraint on it. Running the following query:
```sql
select email, count(*) from ag.ag_login group by email having count(*) > 1;
```
reveals an integrity issue where 14 email addresses are associated with multiple logins.

**Comment by wasade** [comment]
I don't agree. It's useful for subprojects to overload the email field. What's the benefit of the unique constraint?

**Comment by adamrp** [comment]
I didn't know that was useful (can you expand?) and I think there are places in the code base where it is assumed that the email is a unique identifier (e.g., see #471)

**Comment by wasade** [comment]
If the group running the cohort is controlling registration. It was
particularly useful as a redundancy just this past week actually

**Comment by adamrp** [comment]
I see -- I can see some use in that. But I have a feeling that there are additional places in the code that assume that the email address in the `ag_login` table is a unique identifier. I don't have time to investigate fully right now, but I'll look when I get a moment.

**Comment by adamrp** [comment]
Actually can you expand on one more question? How does the group running the cohort actually associate a new `ag_login_id` with the same email address? I'm not sure I know how to do that from the interface, now that we've moved to using only handout kits.

**Comment by wasade** [comment]
They just register with the same details

**Comment by adamrp** [comment]
Not sure I understand, in the context of #471 (see the `addAGLogin` function linked there, which assumes the email address is a unique identifier)

**Comment by wasade** [comment]
We should probably revise the function? Not sure how we'd deal with the
already duplicated emails?

**Comment by adamrp** [comment]
Yeah I think we need to revise the function if we want to retain the possibility of having multiple emails associated with a single ag_login_id. I still don't understand is how this happened in the first place.
